src/gui/accountsettings.cpp

```diff
@@ -339,21 +342,33 @@ void AccountSettings::slotMarkSubfolderEncrypted(FolderStatusModel::SubFolderInf
 
     const auto folder = folderInfo->_folder;
     Q_ASSERT(folder);
+    const auto path = folderInfo->_path;
+    const auto fileId = folderInfo->_fileId;
+    const auto encryptFolder = [this, fileId, folder, path] {
```

Comment:
@FlexW Since you are already capturing 3 members from `folderInfo`, I wonder, why you don't catch the entire pointer?

Reply:
Because the folderInfo pointer is an internal pointer of the list model and is subject of change if the list of folders updates. Thats the root reason for the crash.